ca_description/urdf/create_base.urdf.xacro

```diff
@@ -0,0 +1,237 @@
+<?xml version="1.0" ?>
+<robot xmlns:xacro="http://ros.org/wiki/xacro">
+  <xacro:macro name="create_wheel" params="prefix y_offset wheel_radius wheel_width">
+    <link name="${prefix}_wheel_link">
+    <inertial>
+      <origin xyz="0 0 0" />
+      <mass value="0.01" />
+      <inertia ixx="0.001" ixy="0.0" ixz="0.0" iyy="0.001" iyz="0.0" izz="0.001" />
+    </inertial>
+
+    <visual>
+      <origin xyz="0 0 0" rpy="0 ${PI/2} ${PI/2}" />
+      <geometry>
+        <cylinder radius="${wheel_radius}" length="${wheel_width}" />
+      </geometry>
+    </visual>
+
+    <collision>
+      <origin xyz="0 0 0" rpy="0 ${PI/2} ${PI/2}" />
+      <geometry>
+        <cylinder radius="${wheel_radius}" length="${wheel_width}" />
+      </geometry>
+    </collision>
+    </link>
+
+   <joint name="${prefix}_wheel_joint" type="continuous">
+      <origin xyz="0 ${y_offset} 0.015" rpy="0 0 0" />
+      <parent link="base_link" />
+      <child link="${prefix}_wheel_link" />
+      <axis xyz="0 1 0" />
+    </joint>
+  </xacro:macro>
+
+  <xacro:macro name="create_base" params="diffdrive_update_rate:=40 wheel_separation wheel_diameter:=0.072 wheel_width:=0.024 wheel_torque:=1.0 wheel_accel:=1.8 mass_kg:=3.5 base_diameter *mesh">
+    <xacro:property name="PI" value="3.1415926535897931" />
```

Comment:
According to [documentation](http://wiki.ros.org/xacro) I don't think you can place property tags inside macro tags. It complains in Indigo at least. You can just move the PI property outside I think, and just replace instances of `wheel_radius` with `wheel_diameter / 2` (or maybe better, just make the parameter `wheel_radius` to begin with).